docs-ref-services/latest/servicebus.md

```diff
@@ -18,399 +18,41 @@ Microsoft Azure Service Bus supports a set of cloud-based, message-oriented midd
 * [SDK source code](https://github.com/Azure/azure-sdk-for-python/tree/master/azure-servicebus)
 * [SDK reference documentation](https://docs.microsoft.com/python/api/overview/azure/servicebus/client?view=azure-python)
 
-## What's new in v0.50.0?
+## Libraries for data access
 
-As of version 0.50.0 a new AMQP-based API is available for sending and receiving messages. This update involves **breaking changes**.
+The latest version of the Azure Servicebus libraries is version 7.x.x. We highly recommend using version 7.x.x for new applications.
 
-Please read [Migration from v0.21.1 to v0.50.0](#migration-from-v0211-to-v0500) to determine if upgrading is
-right for you at this time.
+To update existing applications to version 7.x.x, please follow the [migration guide](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/servicebus/azure-servicebus/migration_guide.md)
 
-The new AMQP-based API offers improved message passing reliability, performance and expanded feature support going forward.
-The new API also offers support for asynchronous operations (based on asyncio) for sending, receiving and handling messages.
+If you cannot update existing applications to version 7.x.x, then Microsoft recommends using version 0.50.x
```

Comment:
I'm a bit concerned on this one as we now have deprecated v0.50.

and to go even further, should we mention v0.50 in this landing page? v0.50 itself is a preview version.

I'm personally leaning towards not to mention v0.50 in this page.
thoughts? @lmazuel @ramya-rao-a @rakshith91

Reply:
I agree that we should not promote 0.50.3, but with a couple of caveats - 

1. i'm not sure if it's the best idea to hide the documentation of 0.50.x since a lot of users might still be using it.
2. 0.50.x isn't technically deprecated yet - as much as we dont want users to use it anymore, it can still be installed.

Reply:
JavaScript is in the same boat as Python in this case. We have a v1 of the same package for which we released a v7. I see the merit in providing links for both versions, but would advice against recommending the older version. See https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/cae355082de0f63166c28e3fa340e7c8a3e5d42a/docs-ref-services/latest/service-bus.md for how I am planning to do this for JS